src/main/java/dev/socketmods/socketnukes/block/explosive/TNTExplosive.java

```diff
@@ -226,21 +227,20 @@ public InteractionResult use(BlockState state, Level worldIn, BlockPos pos, Play
         }
     }
 
-    @org.jetbrains.annotations.Nullable
     @Override
-    public BlockEntity newBlockEntity(BlockPos blockPos, BlockState state) {
-        return new ExplosiveTileEntity(blockPos, state);
+    public RenderShape getRenderShape(BlockState p_49232_) {
+        return RenderShape.MODEL;
     }
 
     @org.jetbrains.annotations.Nullable
     @Override
-    public <T extends BlockEntity> BlockEntityTicker<T> getTicker(Level p_153212_, BlockState p_153213_, BlockEntityType<T> p_153214_) {
-        return null;
+    public BlockEntity newBlockEntity(BlockPos blockPos, BlockState state) {
+        return new ExplosiveTileEntity(blockPos, state);
     }
 
     @org.jetbrains.annotations.Nullable
     @Override
-    public <T extends BlockEntity> GameEventListener getListener(Level p_153210_, T p_153211_) {
+    public <T extends BlockEntity> BlockEntityTicker<T> getTicker(Level p_153212_, BlockState p_153213_, BlockEntityType<T> p_153214_) {
```

Comment:
No longer need to import `net.minecraft.world.level.gameevent.GameEventListener`